tests/test_data/acceptance_test_data/network.ini

```diff
@@ -30,6 +30,7 @@ hazard_id = None
 hazard_field_name = waterdepth
 aggregate_wl = max
 hazard_crs = EPSG:4326
+scenario_cost = 0,168789.00
```

Comment:
this can be "dangerous" given that some people have a different number standard using `,` for the floating decimal separator. For now should be alright, but we should be quite clear about the formatting. otherwise I'd suggest using a semicolon `;` as a separator.

Reply:
Adjusted to using ";" as delimiter